hudi-support/extensions/src/main/java/io/onetable/hudi/sync/OneTableSyncTool.java

```diff
@@ -57,9 +56,9 @@ public OneTableSyncTool(Properties props, Configuration hadoopConf) {
 
   @Override
   public void syncHoodieTable() {
-    List<TableFormat> formatsToSync =
+    List<String> formatsToSync =
         Arrays.stream(config.getString(OneTableSyncConfig.ONE_TABLE_FORMATS).split(","))
-            .map(format -> TableFormat.valueOf(format.toUpperCase()))
+            .map(format -> String.valueOf(format.toUpperCase()))
```

Comment:
This can be simplified to `format.toUpperCase` right? it should already be a string

Reply:
done.